### DISCUSSION TITLE
Very stupid question: HOW DO I USE THE TEMPLATES?

### DISCUSSION THREAD

**Discussion by Fontanapink** [windingwind/zotero-better-notes#596] [Q&A] [answered]
I feel like this is a really stupid question, but been trying for the past hour to add items using templates into my workspace note and it is not working! It says, for example: "Template TableView Error: ReferenceError: topItem is not defined".

How do I define @topItem? I've been trying so many different things but none work! And the video in the documentation shows options from the Edit submenu that I don't have in my zotero

**Comment by LynnXie00** [comment, marked as answer]
Check my template. It's working now. I gave screenshots of "how to use it" also.  
https://github.com/windingwind/zotero-better-notes/discussions/406

**Reply by Fontanapink** [reply]
Thank you, this makes everything so much clearer. I hope this works it's way into BN's documentation, as I couldn't find anywhere that you, for example, have to add the word `[Item]` before the template's name for it to work. Thank you so much!

**Reply by windingwind** [reply]
No. It's explicitly required that a template name should start with the quoted item type: https://github.com/windingwind/zotero-better-notes/blob/master/docs/about-note-template.md#note-template-structure
